src/utstring.h

```diff
@@ -121,6 +121,15 @@ do {                                                             \
 
 #define utstring_body(s) ((s)->d)
 
+static char* utstring_release(UT_string *s) {
```

Comment:
If this isn't going to be a macro, I think it needs the word `_UNUSED_` in front of it.
No comment on the functionality; I'll let @troydhanson take it or leave it.